### DISCUSSION TITLE
MAUI samples

### DISCUSSION THREAD

**Discussion by LukasKuchta** [shinyorg/shiny#1093] [Q&A] [answered]
Hi, are there any examples in MAUI platform? I tried BLE hosting sample but it seems to be only x.Forms ready. I'm not able to registre services mannualy. Still missing references. I have installed core and ble hosting. But IPlatfrom not found. 

![image](https://user-images.githubusercontent.com/105702630/189518261-198f86e2-7b10-40a0-900d-5310e2b14c79.png)

Thanks for advice.

**Comment by aritchie** [comment, marked as answer]
No.  v3 which covers MAUI is in an ALPHA version so I haven't had the chance to move everything in samples yet.  For now, you can watch:

http://shinylib.net/release-notes/mobile/vnext.html
https://github.com/shinyorg/shiny/tree/master/samples
